core/core_bind.cpp

```diff
@@ -55,8 +55,11 @@ Error ResourceLoader::load_threaded_request(const String &p_path, const String &
 ResourceLoader::ThreadLoadStatus ResourceLoader::load_threaded_get_status(const String &p_path, Array r_progress) {
 	float progress = 0;
 	::ResourceLoader::ThreadLoadStatus tls = ::ResourceLoader::load_threaded_get_status(p_path, &progress);
-	r_progress.resize(1);
-	r_progress[0] = progress;
+	// Default array should never be modified, it causes hash of the method to change.
+	if (r_progress != DEFVAL(Array())) {
```

Comment:
Why the use of `DEFVAL` here? It's just a helper macro it has no special meaning